### ISSUE TITLE
Whisper volume does not apply to sound effects/multimedia or audio output components set to the Voice AudioTypeGroup

### ISSUE THREAD

**Issue by shiftyscales** [Yellow-Dog-Man/Resonite-Issues#1779]
### Describe the bug?

In prior versions of Resonite- I recall that the whisper mode volume would lower **all** sounds/audio outputs beyond the range of the whisper bubble. This made the whisper bubble useful as an accessibility tool to escape from overstimulating situations.

Previously all audio, including multimedia/videos, etc. would be lowered alongside user voices.

**This functionality has regressed.**

Currently it only seems to apply to user voice streams from actual users. AudioOutputs of any AudioTypeGroup, including the `Voice` group are not lowered as expected.

### To Reproduce

Attach an AudioClipPlayer with a clip, e.g. SineWaveClip to an AudioOutput and attempt to lower its volume by use of the whisper bubble.

Try setting the audio output to various AudioTypeGroup, and note that the volume is never lowered.

### Expected behavior

The whisper bubble should be lowering all audio sources from outside of the bubble per the setting's description:
![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/54213390/bb4b9286-dc36-4a24-bccd-202a06c4ddfb)

### Screenshots

https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/54213390/422162ba-16e2-43ce-b67a-5d0b618e6c90

### Resonite Version Number

2024.4.18.1398

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

I don't expect there to be any relevant details in my log file.

[SHIFTY-LAPTOP - 2024.4.18.1398 - 2024-04-19 08_18_58.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/15046028/SHIFTY-LAPTOP.-.2024.4.18.1398.-.2024-04-19.08_18_58.log)

### Additional Context

This was discovered while I was typing out a response to @HamoCorp in #26 recommending the use of the whisper bubble to attenuate all sounds in the world, including sound effects, multimedia, etc.

Created this issue after having investigated this issue further following my comment in #26.

### Reporters

@shiftyscales

**Comment by Frooxius** [comment]
Fixed this in 2024.4.24.33, thanks!
